pkg/tfvars/vsphere/vsphere_test.go

```diff
@@ -83,24 +93,65 @@ func createControlPlaneConfigs(ipTypes int64) []*v1beta1.VSphereMachineProviderS
 			},
 		}
 
+		var gateway string
 		if ipTypes&ipv4 != 0 {
-			machine.Network.Devices[0].Gateway = ipv4Gateway
-			machine.Network.Devices[0].Nameservers = append(machine.Network.Devices[0].Nameservers, "8.8.8.8")
+			gateway = ipv4Gateway
+			spec.Network.Devices[0].Nameservers = append(spec.Network.Devices[0].Nameservers, "8.8.8.8")
 		} else if ipTypes&ipv6 != 0 {
-			machine.Network.Devices[0].Gateway = ipv6Gateway
-			machine.Network.Devices[0].Nameservers = append(machine.Network.Devices[0].Nameservers, "2001::100")
+			gateway = ipv6Gateway
+			spec.Network.Devices[0].Nameservers = append(spec.Network.Devices[0].Nameservers, "2001::100")
 		}
 
+		idx := 0
 		if ipTypes&ipv4 != 0 {
-			machine.Network.Devices[0].IPAddrs = append(machine.Network.Devices[0].IPAddrs, fmt.Sprintf(ipv4Template, i))
+			ipAddress := ipamv1.IPAddress{}
+			ipAddress.Name = fmt.Sprintf("%s-claim-%d-%d", machine.Name, 0, idx)
+			pool := v1beta1.AddressesFromPool{
+				Group:    "installer.openshift.io",
+				Resource: "IPPool",
+				Name:     fmt.Sprintf("default-%d", idx),
+			}
```

Comment:
This `installer.openshift.io` pool would not exist (by design); at a minimum we will document this, but need to consider/review a bit further for possible side effects.

Reply:
So far I have not seen anything negative.  We do have a card to follow up on this.